src/main/java/core/selenium/WebDriverConfig.java

```diff
@@ -0,0 +1,44 @@
+package core.selenium;
+
+import core.selenium.utils.ReaderFile;
+
+/**
+ * WebDriverConfig class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public final class WebDriverConfig {
+
+    private String nameBrowser;
+
+    private static WebDriverConfig webDriverConfig;
+
+    /**
+     * Constructor of WebDriverConfig class.
+     */
+    private WebDriverConfig() {
+        nameBrowser = ReaderFile.getProperties().getProperty("name_browser");
```

Comment:
in the class ReadFile you must be change the name the method getProperties to getInstance.

Reply:
Thank you so much, this wrong was fixed.

Reply:
Thank you, this wrong was fixed.